### DISCUSSION TITLE
Multiple images of same subject for preventing false detection

### DISCUSSION THREAD

**Discussion by SarimZH** [exadel-inc/CompreFace#750] [Q&A]
Hi there, is it possible to add multiple images of one subject for preventing/reducing false detection cases? I've added multiple images but when I see the list of added subjects through get_all_subjects api I see single instances, entries of each subject.

**Comment by SarimZH** [comment]
Im basically trying to reduce the similarity index in false detection. It is above 90% and in between 90%-95% in various cases.

**Comment by pospielov** [comment]
To see all saved examples of the subject, you can use this endpoint:
https://github.com/exadel-inc/CompreFace/blob/master/docs/Rest-API-description.md#list-of-all-saved-examples-of-the-subject
Not sure that it will help. Unfortunately, the model shows different accuracy on different races.

**Reply by martinenkoEduard** [reply]
Btw what number of classes in your model?
Did you train it yourself?

**Reply by pospielov** [reply]
CompreFace reuses models from FaceNet and InsightFace libraries
Not sure I understand what you mean by a number of classes.
If you mean number of unique people in dataset - then probably you need to check what datasets were used to train those models
